pkg/apis/triggers/v1alpha1/trigger_binding_types.go

```diff
@@ -0,0 +1,63 @@
+package v1alpha1
+
+/*
+Copyright 2019 The Tekton Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Check that TriggerBinding may be validated and defaulted.
+//var _ apis.Validatable = (*TriggerBinding)(nil)
+
+// +k8s:deepcopy-gen=true
+type TriggerBindingSpec struct {
+	Templates []TriggerTemplateRef `json:"templates,omitempty"`
+}
+
+// +k8s:deepcopy-gen=true
+type TriggerTemplateRef struct {
+	TemplateRef string  `json:"templateref,omitempty"`
+	Event       string  `json:"event,omitempty"`
```

Comment:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/432502/60982253-c3674600-a305-11e9-8e13-1e2eab6fd925.png">

in the design we have this as a structure with multiple fields (I think that assume we might want to add more stuff in here, not sure) - happy to go with whatever the majority of us think makes sense

Reply:
no filter? I always thought it is.
I saw the `EventListener` could binding multiple `TriggerBinding` and `TriggerBinding` binding multiple `TriggerTemplate`, 
so I thought there will pick up some one to handle the trigger  based on `event` attribute.
no filter here means we expect the event received should be filter before arrive here? something like `knative eventing`?

Reply:
yeah maybe this would be the right place to put a filter! we do want to incorporate "conditions" (from https://github.com/tektoncd/pipeline/issues/27) eventually 

so @iancoffey i do think we should make this a field with multiple values, at least for now - so we can add more later :D

Reply:
Oh great catch, yes will do.